telepathy-glib: 0.24.1 -> 0.24.2

r-ryantm opened the pull request:
Automatic update generated by [nixpkgs-update](https://github.com/ryantm/nixpkgs-update) tools. This update was made based on information from https://repology.org/metapackage/telepathy-glib/versions.


meta.description for telepathy-glib is: ""

meta.homepage for telepathy-glib is: "https://telepathy.freedesktop.org"

meta.changelog for telepathy-glib is: ""


###### Updates performed
- Version update

###### To inspect upstream changes





###### Impact

<details>
<summary>
<b>Checks done</b> (click to expand)
</summary>

---

- built on NixOS
- The tests defined in `passthru.tests`, if any, passed

- 0 of 0 passed binary check by having a zero exit code.
- 0 of 0 passed binary check by having the new version present in output.
- found 0.24.2 with grep in /nix/store/5bq60zf1z8zzcrjdwzgj4jk97wqn8kcd-telepathy-glib-0.24.2
- directory tree listing: https://gist.github.com/762a777d065db34a80632810aaee27d7

- du listing: https://gist.github.com/d0da246e4ce0f811dc24b7a25a66de58

---

</details>
<details>
<summary>
<b>Rebuild report</b> (if merged into master) (click to expand)
</summary>

```
50 total rebuild path(s)

16 package rebuild(s)

16 x86_64-linux rebuild(s)
16 i686-linux rebuild(s)
2 x86_64-darwin rebuild(s)
16 aarch64-linux rebuild(s)


First fifty rebuilds by attrpath
empathy
gnome3.empathy
gnome3.polari
libsForQt5.telepathy
libsForQt512.telepathy
libsForQt514.telepathy
libsForQt515.telepathy
plasma5Packages.telepathy
telepathy-farstream
telepathy-gabble
telepathy-glib
telepathy-haze
telepathy-idle
telepathy-logger
telepathy-mission-control
telepathy-salut
```

</details>

<details>
<summary>
<b>Instructions to test this update</b> (click to expand)
</summary>

---

Either **download from Cachix**:
```
nix-store -r /nix/store/5bq60zf1z8zzcrjdwzgj4jk97wqn8kcd-telepathy-glib-0.24.2 \
  --option binary-caches 'https://cache.nixos.org/ https://nix-community.cachix.org/' \
  --option trusted-public-keys '
  nix-community.cachix.org-1:mB9FSh9qf2dCimDSUo8Zy7bkq5CX+/rkCWyvRCYg3Fs=
  cache.nixos.org-1:6NCHdD59X431o0gWypbMrAURkbJ16ZPMQFGspcDShjY=
  '
```
(The Cachix cache is only trusted for this store-path realization.)
For the Cachix download to work, your user must be in the `trusted-users` list or you can use `sudo` since root is effectively trusted.

Or, **build yourself**:
```
nix-build -A telepathy-glib https://github.com/r-ryantm/nixpkgs/archive/28d3f54816c5e6dd412460b5f8baf24afc467e6d.tar.gz
```

After you've downloaded or built it, look at the files and if there are any, run the binaries:
```
ls -la /nix/store/5bq60zf1z8zzcrjdwzgj4jk97wqn8kcd-telepathy-glib-0.24.2
ls -la /nix/store/5bq60zf1z8zzcrjdwzgj4jk97wqn8kcd-telepathy-glib-0.24.2/bin
```

---

</details>
<br/>



### Pre-merge build results

We have automatically built all packages that will get rebuilt due to
this change.

This gives evidence on whether the upgrade will break dependent packages.
Note sometimes packages show up as _failed to build_ independent of the
change, simply because they are already broken on the target branch.

Result of `nixpkgs-review` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>2 packages marked as broken and skipped:</summary>
  <ul>
    <li>telepathy-salut</li>
    <li>telepathy_salut</li>
  </ul>
</details>
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li>empathy</li>
    <li>gnome3.polari</li>
    <li>telepathy_qt5 (libsForQt5.telepathy)</li>
    <li>libsForQt512.telepathy</li>
    <li>libsForQt514.telepathy</li>
    <li>telepathy-farstream (telepathy_farstream)</li>
    <li>telepathy-gabble (telepathy_gabble)</li>
    <li>telepathy-glib (telepathy_glib)</li>
    <li>telepathy-haze (telepathy_haze)</li>
    <li>telepathy-idle (telepathy_idle)</li>
    <li>telepathy-logger (telepathy_logger)</li>
    <li>telepathy-mission-control (telepathy_mission_control)</li>
  </ul>
</details>

---

###### Maintainer pings

r-rmcgibbo commented:
Result of `nixpkgs-review pr 115620` at 28d3f548 run on aarch64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>telepathy-salut</li>
  </ul>
</details>
<details>
  <summary>12 packages built:</summary>
  <ul>
    <li><a href="https://gist.github.com/106b2293b4cee57a7e33e26990ec0a50">empathy</a></li> (gnome3.empathy)</li>
    <li><a href="https://gist.github.com/1ab55b1a82c388ebc73efd5960c17089">gnome3.polari</a></li></li>
    <li><a href="https://gist.github.com/d0177233ba5619f6ec80765d8b865c6c">libsForQt5.telepathy</a></li> (libsForQt515.telepathy ,plasma5Packages.telepathy)</li>
    <li><a href="https://gist.github.com/51ff863d5a61daab7b34f2d99b2564cf">libsForQt512.telepathy</a></li></li>
    <li><a href="https://gist.github.com/7d62dcc0bb38aa747f9818b8df13b7d6">libsForQt514.telepathy</a></li></li>
    <li><a href="https://gist.github.com/133b94167a998c8033a981f975a84655">telepathy-farstream</a></li></li>
    <li><a href="https://gist.github.com/c20c43062226e51736f1808db46d4ff0">telepathy-gabble</a></li></li>
    <li><a href="https://gist.github.com/c25aa9612e3ecbc08691ef5d4c4f0066">telepathy-glib</a></li></li>
    <li><a href="https://gist.github.com/baa7ca22c6d7ef476121dd9f3a179ef5">telepathy-haze</a></li></li>
    <li><a href="https://gist.github.com/91b8438c32ea4511435c7864e1cb4610">telepathy-idle</a></li></li>
    <li><a href="https://gist.github.com/d8b6bfaddac9123af6aaba0464644c36">telepathy-logger</a></li></li>
    <li><a href="https://gist.github.com/cfb173403edca7308d1898b19c734ff2">telepathy-mission-control</a></li></li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/maintainers-missing.md">maintainers-missing</a>

Package does not have a maintainer. Consider adding yourself?

Near pkgs/development/libraries/telepathy/glib/default.nix:31:3:
```
   |
31 |   meta = with lib; {
   |   ^
```
</li>
  </ul>
</details>

jtojnar committed:
telepathy-glib: do not propagate python

Python is only used as build tool these days so it is not necessary to ensure consistency across telepathy packages.

jtojnar committed:
telepathy-glib: 0.24.1 -> 0.24.2

Switched to python3 & cleaned up the expression while at it.

https://github.com/TelepathyIM/telepathy-glib/compare/telepathy-glib-0.24.1...telepathy-glib-0.24.2
